README.md

```diff
@@ -30,6 +30,7 @@ Please help me grow this list :) You could:
 - :boot: [Monica](https://www.monicahq.com/), [on GitHub](https://github.com/monicahq/monica): CRM for personal relationships.
 - :boot: [NextCloud](https://nextcloud.com/), [on GitHub](https://github.com/nextcloud): self-hosted personal cloud solution (file syncing, agenda, contact, etc.)
 - :boot: [Open Whisper Systems](https://www.signal.org/), [on GitHub](https://github.com/signalapp): creating Signal, a fast, simple, and secure communication app.
+- :moneybag: [Puri.sm](https://puri.sm/), [Cogs](https://code.puri.sm/), [on GitHub](https://github.com/purism): High-quality laptops that protect your freedom and privacy.
```

Comment:
To follow the pattern used in the other line, can you:
- replace `[Cogs](https://code.puri.sm/)` by `[on their Cogs](https://code.puri.sm/)`
- add the funding amount with a link to crunchbase just after the link? e.g.: `[$5.6M](https://www.crunchbase.com/organization/purism)`

Thank you :pray: